clickstream-event-visualiser-ui/src/main/java/clickstream/event_visualiser_ui/ui/CSEvListItem.kt

```diff
@@ -0,0 +1,7 @@
+package clickstream.event_visualiser_ui.ui
+
+public data class CSEvListItem(
```

Comment:
how about `CSEvItem`?

Reply:
Since item is a part of List in UI, i added List. Just a convention that i follow.

Reply:
in that case, i believe we can have inner data class